src/platform/src/common/components/Organization/OrganizationSettingsContainer.jsx

```
@@ -0,0 +1,184 @@
+            };
+          case 'file-too-large':
+            return {
+              message: 'File too large. Please upload images smaller than 5MB.',
+              type: 'error',
+            };
+          case 'validation-error':
+            return {
+              message: 'Please fix the validation errors and try again.',
+              type: 'warning',
+            };
+          case 'error':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)